docs/ml/deploy/tflite_cpu.md

```diff
@@ -38,15 +38,17 @@ You can choose to configure your service with an existing model on the machine o
 1. To configure your service and deploy a model onto your machine, select **Deploy Model On Robot** for the **Deployment** field.
 
 2. Click on **Models** to open a dropdown with all of the ML models available to you privately, as well as all of the ML models available in [the registry](https://app.viam.com), which are shared by users.
+   Models that your organization has trained that are not uploaded to the registry will appear first in the dropdown.
    You can select from any of these models to deploy on your robot.
+   Only TensorFlow Lite models are shown.
 
 {{<imgproc src="/services/deploy-model-menu.png" resize="700x" alt="Models dropdown menu with models from the registry.">}}
 
 {{% alert title="Tip" color="tip" %}}
 To see more details about a model, open its page in [the registry](https://app.viam.com).
 {{% /alert %}}
 
-3. Also, optionally select the **Number of threads**.
+1. Also, optionally select the **Number of threads**.
```

Comment:
3 is correct. The image and the tip should be indented. Can you see if you can fix that?

Reply:
let me know if you need help

Reply:
I think I've fixed it now, turned off a markdown ordered list setting in vscode